components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/ClaimMetadataManagementService.java

```diff
@@ -162,6 +171,29 @@ default void removeAllClaims(int tenantId) throws ClaimMetadataException {
 
     }
 
+    /**
+     * Update attribute mappings for local claims in bulk.
+     *
+     * @param localClaimList    list of local claims.
+     * @param tenantDomain      tenant domain name.
+     * @param userStoreDomain   user store domain name.
+     * @throws ClaimMetadataException if error occurs while updating local claims.
```

Comment:
Descriptions should be in sentence case.

Reply:
Addressed with commit 2e3f3c951772d195bcbedb1a7b81f4815b531259 in https://github.com/wso2/carbon-identity-framework/pull/3774 PR